native/dlaqr1.go

```diff
@@ -0,0 +1,56 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlaqr1 sets v to a scalar multiple of the first column of the product
+//  (H - (sr1 + i*si1)*I)*(H - (sr2 + i*si2)*I)
+// where H is a 2×2 or 3×3 matrix and i is the imaginary unit. Scaling is done
+// to avoid overflows and most underflows.
```

Comment:
Could you add that I is the identity matrix? Kind of obvious, but somewhat tricky with all the i/I